packages/dropdownMenu/tests/DropdownMenu.test.tsx

```diff
@@ -255,22 +255,18 @@ describe("Dropdown", () => {
     );
     const trigger = component.find(`#${triggerId}`);
 
-    trigger.simulate("focus");
-    trigger.simulate("keyDown", {
-      key: " " // space bar
-    });
     expect(onSelectFn).not.toHaveBeenCalled();
     trigger.simulate("keyDown", {
       key: "ArrowDown"
     });
     trigger.simulate("keyDown", {
       key: "Enter"
     });
-    trigger.simulate("blur");
 
     // using `expect.anything()` because the second parameter is an object
     // that comes from Downshift, and there's no good way to know exactly
     // what to expect
     expect(onSelectFn).not.toHaveBeenCalledWith("edit", expect.anything());
+    expect(onSelectFn).toHaveBeenCalled();
```

Comment:
Added this expect as the original expect statement is a false-positive test.

The edit menu item in this test is disabled and cannot be selected by the user. I think we could break this out into another test where we only have disabled menu-items and test for `onSelectFn.not.toHaveBeenCalled()`.

Reply:
In addition to the above, we could modify this to "selects second item and passes it's value to onSelect".